parts/dcoscustomdata173.t

```diff
@@ -346,3 +346,7 @@ write_files:
   path: "/opt/azure/containers/provision.sh"
   permissions: "0744"
   owner: "root"
+- path: /var/lib/dcos/mesos-slave-common
```

Comment:
this is all it takes? You just put the map into `mesos-slave-common` and it knows to apply those labels to the nodes?